lib/TransferConfigurations.cs

```diff
@@ -48,16 +53,22 @@ public TransferConfigurations()
             this.parallelOperations = Environment.ProcessorCount * 8;
             this.MemoryChunkSize = Constants.DefaultMemoryChunkSize;
 
+#if DOTNET5_4
```

Comment:
The original code use 6 and 2 or 6 and 4 values which I simplified to 6 and 2. I hope that's not a problem.

Reply:
Please keep existing value, Emma may have further suggestion in this part. The value was chose based on perf testing.

---
In reply to: [452238119](https://github.com/Azure/azure-storage-net-data-movement/pull/226#discussion_r452238119) [](ancestors = 452238119)

Reply:
Preserving current values will require me to make the value nullable.  This makes the code harder to follow but I'm happy to make this change.

As a side note, it would good to have a comment there or a link that provides more context regarding those magic values.

Reply:
That's done now @jiacfan